extensions/metadata_exchange/plugin.cc

```diff
@@ -130,7 +132,13 @@ bool PluginRootContext::updatePeer(std::string_view key,
     }
   }
 
-  auto bytes = Base64::decodeWithoutPadding(peer_header);
+#ifndef NULL_PLUGIN
+  auto peer_header_view = peer_header;
+#else
+  auto peer_header_view = Wasm::Common::toAbslStringView(peer_header);
```

Comment:
I am not fully sure. Should `Wasm::Common::toAbslStringView` wraps NULL_PLUGIN? Otherwise, Wasm::Common::toAbslStringView is duplicating the envoy version

Reply:
Not sure how could you wrap this?

Reply:
NVM